_applications/equoid.adoc

```diff
@@ -0,0 +1,206 @@
+
+In addition, you can add a frequent item by clicking on the add item button. Finally, you can add another data handler via the command line, a script is included in equoid-openshift to facilitate additional data handler instances. Once these are up and running, the web UI will automatically add their data in a separate graph tab. 
+ 
+=== Equoid Application Settings
+We provide a number of variables to modify per the needs of end users. In this section, these are described. 
+
+==== Data Publisher
+
+`DATA_URL_PRIMARY` - URL of either full dataset or list of all field values for the linear and single `OP_MODE` settings, respectively.
+
+`DATA_URL_SECONDARY` - URL of all field values for the dual `OP_MODE` setting.
+
+`OP_MODE` - Operating mode: single for a list of field values from which random elements will be generated, dual for two lists of field values from which random elements will be generated, linear for a dataset meant to be read in sequence and transmitted.
+
+==== Data Handler
+
+`WINDOW_SECONDS` - Size, in seconds of window for which the top-k elements should be determined. 
+
+`SLIDE_SECONDS` - Size, in seconds, of the amount to slide the sample window by each iteration. 
+
+`BATCH_SECONDS` - Size, in seconds, of the batch size to be acquired from the broker.
+
+`OP_MODE` - As with the data publisher, single for a list of field values from which random elements will be generated, dual for two lists of field values from which random elements will be generated, linear for a dataset meant to be read in sequence and transmitted.
+
+==== Settings for Examples
+
+Single-field sales messages generated from a set of possible values for item ID as provided by UCI dataset::
+* `DATA_URL_PRIMARY` https://raw.githubusercontent.com/radanalyticsio/equoid-data-publisher/master/data/StockCodes.txt for data publisher 
+* `OP_MODE` single for both data publisher and handler
+
+Dual-field sales messages generated from a set of possible values for item ID and country as provided by UCI dataset::
+* `DATA_URL_PRIMARY` https://raw.githubusercontent.com/radanalyticsio/equoid-data-publisher/master/data/StockCodes.txt for data publisher 
+* `DATA_URL_SECONDARY` https://raw.githubusercontent.com/radanalyticsio/equoid-data-publisher/master/data/Countries.txt for data publisher 
+* `OP_MODE` dual for both data publisher and handler
+
+Single-field sales messages sent in order as provided in the UCI dataset::
+* `DATA_URL_PRIMARY` https://raw.githubusercontent.com/radanalyticsio/equoid-data-publisher/master/data/StockCodesLinear.txt for data publisher 
+* `OP_MODE` linear for both data publisher and handler
+
+[[expansion]]
+== Expansion
```

Comment:
Perhaps parameter tuning could be put in the expansion - at the moment it is mentioned near the beginning of the article, with the tag that "We don't do this there".  (I'm not suggesting you do parameter tuning mind, so if the Expansion section is actually for things you're still working on then definitely ignore!)

Reply:
Good point, will add this.